content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -58,6 +58,61 @@ corresponding reclamation or reconciliation process, as a given admission
 controller does not know for sure that a given request will pass all of the
 other admission controllers.
 
+The ordering of these calls can be seen below.
+
+
+{{< mermaid >}} 
+sequenceDiagram
+
+%% Have to set fontSize globally as this isn't respected due to https://github.com/mermaid-js/mermaid/issues/3990
+%%{init:{"theme":"neutral","fontSize": "22px", "sequence": {"mirrorActors":true},
+    "themeVariables": {
+        "actorBkg":"#326ce5",
+        "actorTextColor":"white",
+        "loopTextColor": "#326ce5",
+        "noteBkgColor": "grey",
+        "fontSize": "22px"
+}}}%%
+
+
+    %% This is the alt-text for accessibility, and should be updated every time the diagram is.
+    accDescr {
+        Sequence diagram for kube-apiserver handling requests during the admission phase showing mutation webhooks, followed by validatingadmissionpolicies and finally validating webhooks. 
+        It shows that the continue until the first rejection, or being accepted by all of them. 
+        It also shows that mutations by mutating webhooks cause all previously called webhooks to be called again.
+    }
+
+    participant User as User 
+    participant API as Kubernetes API Server
+    participant Auth as Authentication + Authorisation
+    %% participant AdmCtrl as Admission Control
+    box Gainsboro  Admission Control <br>Called until first rejection
```

Comment:
1. The [documentation style guide](https://kubernetes.io/docs/contribute/style/style-guide/#language) instructs that we use a U.S. English spelling.
2. The word “until” assumes that it is a looping process and the actual diagram specifies otherwise. I will suggest that you only say “Admission Control”.

```suggestion
    participant Auth as Authentication + Authorization
    %% participant AdmCtrl as Admission Control
    box Gainsboro  Admission Control
```

Reply:
The "until" is referencing the fact that the process stops at the first rejection. What would you suggest as a better way of phrasing this?

Reply:
Either the one I suggested above or the one below.

>  Admission Control <br>Halts after the first rejection